src/flow.h

```diff
@@ -338,6 +338,8 @@ typedef struct Flow_
 
     uint32_t flags;
 
+    uint8_t reverted;
```

Comment:
adding a new (undocumented!) var for setting a flag seems rather excessive. We have f->flags for this.